index.js

```diff
@@ -521,6 +547,16 @@ studio.internal = (function(proto) {
         app.makeGetterRequest(id, 1, 0, true);
       }
     }
+
+    this.async._makeEventRequest = function() {
+      if (eventSubscriptions.length > 0) {
+        var noZeroArray = eventSubscriptions.filter(v => v[1] != undefined);
+        var minTimeStamp = noZeroArray.length > 0 ? Math.min.apply(Math, noZeroArray.map(v => v[1])) : undefined;
```

Comment:
I don't think Math.min is correct here. I would think each call to subscribeToEvents would give history based on the current startingFrom argument not the minimum of all subscribers.

Reply:
thought new protobuf call with new timestamp overrides the old subscription, and thus to serve the old subscriber we must use the min timestamp to cover for all subscribers

Reply:
but changed the code, as i got feedback that new protobuf call with new timestamp does not override it